src/components/Balance.tsx

```diff
@@ -19,19 +20,42 @@ const Balance: React.FC<BalanceProps> = ({
   unit,
   prefix,
   onClick,
+  fixedWidth,
```

Comment:
I want to avoid specific props like this where possible. It would be better if we had a `containerProps` prop and created a new component that composed this one.